### PR TITLE
fix runner re-registration issue

### PR DESCRIPTION
**Pull request by dennybaa**
Fix
```
-----------------------------
 Finish Install Dependencies
-----------------------------

--------------------------------------------------------------------------------
|        ____ _ _   _   _       _          _        _   _                      |
|       / ___(_) |_| | | |_   _| |__      / \   ___| |_(_) ___  _ __  ___      |
|      | |  _| | __| |_| | | | | '_ \    / _ \ / __| __| |/ _ \| '_ \/ __|     |
|      | |_| | | |_|  _  | |_| | |_) |  / ___ \ (__| |_| | (_) | | | \__ \     |
|       \____|_|\__|_| |_|\__,_|_.__/  /_/   \_\___|\__|_|\___/|_| |_|___/     |
|                                                                              |
|                       Self-hosted runner registration                        |
|                                                                              |
--------------------------------------------------------------------------------
Cannot configure the runner because it is already configured. To reconfigure the runner, run 'config.cmd remove' or './config.sh remove' first.
```

### CODE REVIEW

**Comment by dennybaa** [comment]
Howdy @davidkarlsen .
Apparently the auto-upgrades are backed into `install-runner`, this PR won't harm. I've tried to use the upstream as is with `--replace` and it's still has this issue, so I suggest this small PR. Won't harm though.

File: entrypoint.sh [hunk 1]
```diff
@@ -18,5 +18,8 @@ fi
 
 sudo install-runner
 
-./config.sh --unattended --replace --url "${RUNNER_URL}" --token "${RUNNER_TOKEN}"
+# Reconfigure from the clean state in case of runner failures/restarts
+./config.sh remove --token "${RUNNER_TOKEN}"
```
Comment (davidkarlsen) [inline entrypoint.sh:22]: does this succeed if it's not registered?

Comment (dennybaa) [reply entrypoint.sh:22]: Hey David. Yes it does.
```
# Runner removal


√ Runner removed successfully
√ Removed .credentials
√ Removed .runner
....
# Authentication
√ Connected to GitHub
# Runner Registration
√ Runner successfully added
√ Runner connection is good
# Runner settings
√ Settings Saved.
√ Connected to GitHub

Current runner version: '2.284.0'
2021-11-17 20:11:30Z: Listening for Jobs
```

Comment (dennybaa) [reply entrypoint.sh:22]: I can not be sure on a super-long term, I haven't looked into the token and don't know it has expiration time. Though on 6d of run killing pid produces a successful restart as you can see above. Token is updated by the operator, should be... So  worked fine for me.

Comment (davidkarlsen) [reply entrypoint.sh:22]: Hm, I thought `--replace` should do the same trick, but in one command.

**Pull request merged**